system-addon/lib/SectionsManager.jsm

```diff
@@ -41,12 +41,8 @@ const BUILT_IN_SECTIONS = {
       button: {id: options.disclaimer_buttontext || "section_disclaimer_topstories_buttontext"}
     },
     maxRows: 1,
-    availableContextMenuOptions: ["CheckBookmark", "SaveToPocket", "Separator", "OpenInNewWindow", "OpenInPrivateWindow", "Separator", "BlockUrl"],
-    infoOption: {
-      header: {id: options.provider_header || "pocket_feedback_header"},
-      body: {id: options.provider_description || "pocket_feedback_body"},
-      link: {href: options.info_link, id: "section_info_privacy_notice"}
-    },
```

Comment:
Here's where I thought we could configure the "about object" based on topstories.options, which we could then access in section-menu-options.js.